WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -345,6 +384,8 @@ private extension SupportTableViewController {
         static let contactEmail = NSLocalizedString("Contact Email", comment: "Support email label.")
         static let contactEmailAccessibilityHint = NSLocalizedString("Shows a dialog for changing the Contact Email.", comment: "Accessibility hint describing what happens if the Contact Email button is tapped.")
         static let emailNotSet = NSLocalizedString("Not Set", comment: "Display value for Support email field if there is no user email address.")
+        static let wpAccount = NSLocalizedString("WordPress.com Account", comment: "WordPress.com sign-out section header title")
+        static let logOutButtonTitle = NSLocalizedString("Log Out", comment: "Button for confirming logging out from WordPress.com account")
```

Comment:
same consideration here about the strings

Reply:
Same as [this reply](https://github.com/wordpress-mobile/WordPress-iOS/pull/19618#discussion_r1026421774)